src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -215,6 +215,9 @@ def start_job_status_loop(
                     "name": getattr(e, "name", type(e).__name__),
                 }
                 self.send_comm_message(MESSAGE_TYPE["ERROR"], error)
+                # if job init failed, set the lookup loop var back to False and return
+                self._running_lookup_loop = False
```

Comment:
I noticed that the BE would still try to start the job lookup loop even if it had failed to initialise jobs. This restores some more sane behaviour, returning things to the state they were in prior to attempting to init jobs.